src/client.js

```diff
@@ -163,7 +173,7 @@ export default class StitchClient {
         useRefreshToken: true,
         rootURL: this.rootURLsByAPIVersion[v2][API_TYPE_CLIENT]
       }
-    ).then(() => this.auth.clear());
+    ).then(async () => await this.auth.clear()); // eslint-disable-line space-before-function-paren
```

Comment:
i don't think this line needs to change, right?
I think using async/await when you're already inside a promise chain `then()` is probably not optimal, since that will get transpiled into a whole new promise chain.